src/main/java/com/jagrosh/jmusicbot/settings/QueueType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.settings;
+
+/**
+ *
+ * @author Wolfgang Schwendtbauer
+ */
+public enum QueueType
+{
+    LINEAR("\u23E9", "Linear"),       // ⏩
+    FAIR("\uD83D\uDD22", "Fair"),     // 🔢
+    RANDOM("\uD83D\uDD00", "Random"); // 🔀
```

Comment:
Seems like this random queue was unintentionally left in? There's no random queue implemented yet